keras_preprocessing/image/image_data_generator.py

```diff
@@ -943,6 +943,9 @@ def fit(self, x,
             np.random.seed(seed)
 
         x = np.copy(x)
+        if self.rescale:
```

Comment:
Even though somewhat obvious can you add a comment explaining why rescaling before featurewise normalization, etc.. ?

Reply:
In the case of featurewise normalization and ZCA whitening rescaling isn’t really necessary since they apply their own kind of "scaling". So, this bug fix is needed more for consistent behavior no matter whether the `rescale` parameter is specified or not (before the code will produce unexpected results if the `rescale` parameter is specified).

The change to the code is useful when a user wants to apply rescaling with featurewise centering only. With this bug fix the code will produce correct outputs.

Reply:
I meant a comment in the code, explaining to the developer why rescaling before...